### ISSUE TITLE
use expression with external parameters for the gravity-force

### ISSUE THREAD

**Issue by thomas-saigre** [feelpp/feelpp#2309]
Symbols in expression for the argument `gravity-force` are not taken into account.
For instance,

```cfg
[heat-fluid]
gravity-force={9.80665*cos(pi/3*t), 0, 9.80665*sin(pi/3*t)}:t
```
will result in a constant gravity force of `{0, 0, 9.80665}`, independent of `t`.

**Comment by thomas-saigre** [comment]
@vincentchabannes @prudhomm I might have found a workaround consisting in manually updating the gravity force during execution :
```c++
HEATFLUID_CLASS_TEMPLATE_DECLARATIONS
void
HEATFLUID_CLASS_TEMPLATE_TYPE::updateGravityForce( const std::string &gravityStr )
{
    M_gravityForce = expr<nDim,1,2>( gravityStr,"",this->worldComm(),this->repository().expr() );
}
```
The application seems to be working with this, we will see how it goes...

**Comment by vincentchabannes** [comment]
@thomas-saigre yes it can be work but not the good way. We need to add heatfluid in modelphysics and insert gravity forces inside (as done for fluid for example). Gravity expr will be automatically managed (symbols expr and in time). I will add this feature as soon as possible in the branch related to wall shear stress and parallelism.
